src/FacadeAccountCreation/FacadeAccountCreation.Core/Models/Organisations/PaymentCalculationResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace FacadeAccountCreation.Core.Models.Organisations
+{
+    [ExcludeFromCodeCoverage]
+    public class PaymentCalculationResponse
+    {
+        public decimal ProducerRegistrationFee { get; set; }
+        public decimal ProducerOnlineMarketPlaceFee { get; set; }
+        public decimal SubsidiariesFee { get; set; }
+        public decimal TotalFee { get; set; }
+        public decimal PreviousPayment { get; set; }
+        public decimal OutstandingPayment { get; set; }
+        public SubsidiariesFeeBreakdown SubsidiariesFeeBreakdown { get; set; }
+    }
+
+    [ExcludeFromCodeCoverage]
+    public class SubsidiariesFeeBreakdown
+    {
+        public decimal TotalSubsidiariesOMPFees { get; set; }
+        public int CountOfOMPSubsidiaries { get; set; }
+        public decimal UnitOMPFees { get; set; }
+        public FeeBreakdown FeeBreakdowns { get; set; }
```

Comment:
This should be a **collection** of FeeBreakdown's in case the number of subsidiaries is greater than 20.
For example:
NumberOfSubsidiaries      FeeBreakdowns
              54                |       array of 2 FeeBreakdown's for the first 20 subsidiaries and the next 34 subsidiaries
            104                |       array of 3 FeeBreakdown's for the first 20 subsidiaries, the next 80 subsidiaries & 4 subs

Reply:
Done